### PR TITLE
fix(frontend): signIn signUp screens markdown bb-129

### PR DESCRIPTION
**Pull request by ihorLysak**
### Summary
fixed markdown of signIn signUp screens
Screenshot 2024-08-22 at 21 09 03
Screenshot 2024-08-22 at 21 09 14

### CODE REVIEW

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 1]
```diff
@@ -28,51 +29,72 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+
 					<Input
 						control={control}
 						errors={errors}
-						label="Name"
-						name="name"
-						placeholder="name"
-						type="text"
+						label="name"
+						name="email"
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:52]: Now we have 2 fields for email

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 2]
```diff
@@ -28,51 +29,72 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+
 					<Input
 						control={control}
 						errors={errors}
 						label="Password"
 						name="password"
-						placeholder="password"
-						type="password"
+						placeholder="*******"
+						type="text"
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:72]:
```suggestion
						type="password"
```

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 3]
```diff
@@ -28,51 +29,72 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+							Already have an account? Go to{" "}
+							<Link to={AppRoute.SIGN_IN}>Sign in</Link>
+						</span>
+					</div>
+
 					<Input
 						control={control}
 						errors={errors}
-						label="Name"
+						label="name"
```
Comment (ViktoriiaKozopas) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:51]: Why `name` isn't capitalized? 🤔

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 4]
```diff
@@ -28,51 +29,72 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+					<Button label="CREATE AN ACCOUNT" type="submit" variant="dark" />
+				</form>
+			</div>
+
+			<div className={styles["logo-container"]}>
+				<span className={styles["logo"]}>LOGO</span>
+			</div>
+
+			<img
+				alt="ripple-effect-bg"
```
Comment (ViktoriiaKozopas) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:84]: image
Please, update the `alt` to the meaningful text or remove this attribute.

**Review by ViktoriiaKozopas — APPROVED** [review]
⭐

File: apps/frontend/src/pages/auth/auth.tsx [hunk 5]
```diff
@@ -13,7 +11,6 @@ import {
 } from "~/modules/users/users.js";
 
 import { SignInForm, SignUpForm } from "./components/components.js";
-import styles from "./style.module.css";
```
Comment (JD-Rosales) [inline apps/frontend/src/pages/auth/auth.tsx:16]: I think we also need to delete the CSS module file.

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 6]
```diff
@@ -17,7 +18,7 @@ type Properties = {
 const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 	const { control, errors, handleSubmit } = useAppForm<UserSignUpRequestDto>({
 		defaultValues: DEFAULT_SIGN_UP_PAYLOAD,
-		validationSchema: userSignUpValidationSchema,
+		validationSchema: userSignInValidationSchema,
```
Comment (revenets) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:21]: Why do we use sign-in validation schema for sign-up form?

File: apps/frontend/src/pages/auth/auth.tsx (hunk 5 above)
Comment (ihorLysak) [reply apps/frontend/src/pages/auth/auth.tsx:16]: It is already deleted in this PR

**Review by WilhelmWesser — APPROVED** [review]

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 7]
```diff
@@ -53,26 +53,48 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
 						type="text"
 					/>
+
 					<Input
 						control={control}
 						errors={errors}
 						label="Email"
 						name="email"
-						placeholder="email"
+						placeholder="name@gmail.com"
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:62]:
```suggestion
						placeholder="name@example.com"
```

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 8]
```diff
@@ -28,23 +29,22 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+		<div className={styles["container"]}>
+			<div className={styles["form-container"]}>
+				<form className={styles["form"]} onSubmit={handleFormSubmit}>
+					<div className={styles["form-header"]}>
+						<div className={styles["form-header__logo-container"]}>
+							<div className={styles["form-header__logo"]} />
+							<span className={styles["form-header__logo-text"]}>LOGO</span>
+						</div>
+
+						<h1 className={styles["form-header__text"]}>CREATE AN ACCOUNT</h1>
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:41]: Let's use `text-transform: uppercase` here. Same for other similar cases

File: apps/frontend/src/pages/auth/components/sign-up-form/style.module.css [hunk 9]
```diff
@@ -1,33 +1,137 @@
...
+	grid-template-columns: 1fr 1fr;
+	gap: 30px;
+	height: 100dvh;
+	padding: 15px;
+	overflow: hidden;
+	background-color: var(--background);
+}
+
+.form-container {
+	z-index: 1;
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-up-form/style.module.css:13]: Use a variable for z-index

**Review by fshabanov — CHANGES_REQUESTED** [review]
Your PR has too many changes, that shouldn't be there

File: apps/frontend/src/pages/auth/components/sign-up-form/style.module.css [hunk 10]
```diff
@@ -1,33 +1,140 @@
...
+	text-transform: uppercase;
+}
+
+.form-header__text {
+	margin-block: 10px;
+	text-transform: uppercase;
+}
+
+.form-header__sub-text {
+	font-size: medium;
```
Comment (yulyaolshanska) [inline apps/frontend/src/pages/auth/components/sign-up-form/style.module.css:64]: I think you should use	font-size with px

Comment (fshabanov) [reply apps/frontend/src/pages/auth/components/sign-up-form/style.module.css:64]: Agree, also I believe `medium` is the default one

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by fshabanov — APPROVED** [review]

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 11]
```diff
@@ -28,23 +29,22 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+							<div className={styles["form-header__logo"]} />
+							<span className={styles["form-header__logo-text"]}>logo</span>
+						</div>
+
+						<h1 className={styles["form-header__text"]}>create an account</h1>
+						<span className={styles["form-header__sub-text"]}>
+							Already have an account? Go to{" "}
+							<Link to={AppRoute.SIGN_IN}>Sign in</Link>
+						</span>
+					</div>
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:46]: These are not parts of a form, should be outside of `<form>` tag
Should also be fixed for sign-in screen

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by JD-Rosales — APPROVED** [review]

**Review by revenets — APPROVED** [review]

**Review by JD-Rosales — APPROVED** [review]

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx [hunk 12]
```diff
@@ -28,51 +29,67 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
+						<span className={styles["form-header__logo-text"]}>logo</span>
+					</div>
+
+					<h1 className={styles["form-header__text"]}>create an account</h1>
+					<span className={styles["form-header__sub-text"]}>
+						Already have an account? Go to{" "}
+						<Link to={AppRoute.SIGN_IN}>Sign in</Link>
+					</span>
 				</div>
+
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:46]: Not fixed. Should be outside of `<form>` tag. Same for SignInForm

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx (hunk 12 above)
Comment (ihorLysak) [reply apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:46]: So only input fields should be inside of form?

Comment (fshabanov) [reply apps/frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx:46]: That's right. The logo and links are not a part of a form

File: apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx [hunk 13]
```diff
@@ -29,47 +29,41 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
-						name="password"
-						placeholder="*******"
-						type="text"
-					/>
+				<Input
+					control={control}
+					errors={errors}
+					label="Email"
+					name="email"
+					placeholder="name@gmail.com"
```
Comment (yulyaolshanska) [inline apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx:52]: Should be name@example.com

File: apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx [hunk 14]
```diff
@@ -29,47 +29,41 @@ const SignInForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
...
-					<Button label="SIGN IN" type="submit" variant="dark" />
-				</form>
-			</div>
+				<Input
+					control={control}
+					errors={errors}
+					label="Password"
+					name="password"
+					placeholder="*******"
+					type="text"
```
Comment (yulyaolshanska) [inline apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx:62]: Type should be password

File: apps/frontend/src/pages/auth/auth.tsx [hunk 15]
```diff
@@ -46,9 +50,39 @@ const Auth: React.FC = () => {
 
 	return (
 		<>
```
Comment (fshabanov) [inline apps/frontend/src/pages/auth/auth.tsx:52]: I believe this fragment is redundant

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Pull request merged**